build/Dockerfile

```diff
@@ -204,7 +243,7 @@ RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode
 	--mount=type=bind,from=nginx-files,src=app-protect-8.repo,target=/etc/yum.repos.d/app-protect-8.repo \
```

Comment:
@oseoin can line#204 be removed alltogether?

Reply:
This is needed for the WAF+DOS image I think